File: 41.First Missing Positive/solution.py
class Solution(object):
    def firstMissingPositive(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums = sorted(nums)
        i = 1
        for k in nums:
            if k==i:
                i+=1
        return i
        

File: 41.First Missing Positive/solution.cpp
class Solution {
public:
    int firstMissingPositive(vector<int>& num) {
        sort(num.begin(),num.end());
        int n = 1;
        for(int i:num){
            if(i==n){
                n++;
            }
        }
        return n;
    }
};